### DISCUSSION TITLE
run example (deep_cfr_pytorch.py)file, but cannot converge to Nash equilibrium

### DISCUSSION THREAD

**Discussion by alex0zhen** [deepmind/open_spiel#1095] [Q&A]
I run example file  (deep_cfr_pytorch.py),
set deep_cfr parameter : num_iterations=400, num_traversals=40...
kuhn_poker game cannot converge to Nash equilibrium
how to set some parameter to converge?

**Comment by lanctot** [comment]
Oh sorry, I completely missed this. Can you please double check your hyperparameyters coincide with the ones in this paper? https://arxiv.org/abs/2103.00187

Also can you show us your curves?

**Comment by lanctot** [comment]
Did you try running this example? https://github.com/deepmind/open_spiel/blob/master/open_spiel/python/examples/deep_cfr_pytorch.py

It was working last time we checked, but that was almost two years ago.
